ui/components/MeshModelRegistry/MeshModelComponent.js

```diff
@@ -334,8 +334,8 @@ const MeshModelComponent = ({ modelsCount, componentsCount, relationshipsCount }
             elevation={3}
             className={`${StyleClass.cardStyle} ${animate ? StyleClass.cardStyleAnimate : ''}`}
             style={{
-              backgroundColor: `${view === MODELS && animate ? 'white' : ''}`,
-              color: `${view === MODELS && animate ? 'black' : ''}`,
+              backgroundColor: `${view !== MODELS && animate ? '#677a84' : ''}`,
+              color: `${view !== MODELS && 'white'}`,
```

Comment:
Can we please consider using our theme? 

You can define these new colors in our theme and refer them as variables.

Reply:
Yes Sure.

Reply:
@theBeginner86 I changes all the color codes to theme references.

Reply:
Thank you!